### PR TITLE
Improve German translations

### PR DESCRIPTION
**Pull request by salim-b**
(Native speaker here)

### CODE REVIEW

**Comment by 12people** [comment]
@salim-b 
Thanks for the contribution!
Just one thing that I'd like to sort out: "Export" is meant as a verb in the original — it's used for buttons in the interface. I would say that should still be "Exportieren". There's a corresponding "Importieren" too, which you left untouched.
What do you think?

**Comment by salim-b** [comment]
Ah, I've overlooked `Importieren`. I've simply changed the verb `Exportieren` to the noun `Export` to match the other entries in the same menu which are also nouns (`Projekte`, `Berichte`, `Einstellungen`, ...).

Now that I had a closer look, I see that `Importieren` is used in the "database icon menu" displayed on the export screen... I think in that particular menu it's better to use verbs than nouns. But at the same time I think that menu is totally weird UX-wise (nobody would search for the *import* functionality on the *export* screen...).

tl;dr: I reverted the translation to the verb `Exportieren` (plus added another minor change I had overlooked before). Coming up with a better import/export UX would better be tackled in a different issue, I guess.

**Comment by 12people** [comment]
Great, thanks for the fix!

Yeah, UX-wise, the Export functionality is less than ideal. Aside from the less discoverable "Import" button on the Export screen, there's also two buttons labeled "Export" that actually do two different things (one is a database export, another is a file export).

Definitely something worth tackling in the future, though I can't promise it'll be me who tackles it.
